tests/src/cgeo/geocaching/location/GeoPointParserTest.java

```diff
@@ -313,4 +308,23 @@ public void test6802() {
         }
     }
 
+    @Test
+    public void test8078() {
+        try {
+            GeopointParser.parse("2.2.3.8");
+            failBecauseExceptionWasNotThrown(Geopoint.ParseException.class);
+        } catch (final Geopoint.ParseException e) {
+            // expected
+        }
```

Comment:
this can be replaced by 
`        assertParsingFails("2.2.3.8"); `

Reply:
Done.